src/components/NoteDialogComponents/NoteDialogNoteComponents/NoteDialogNoteContent.tsx

```diff
@@ -26,21 +28,32 @@ export const NoteDialogNoteContent: FC<NoteDialogNoteContentProps> = ({noteId, a
     dispatch(Actions.onNoteBlur());
   };
 
+  const isImage = useImageChecker(text);
+
   return (
     <div className="note-dialog__note-content">
-      <textarea
-        className="note-dialog__note-content__text"
-        disabled={!editable}
-        onBlur={(e) => onEdit(noteId!, e.target.value ?? "")}
-        onFocus={onFocus}
-        onKeyDown={(e) => {
-          if (e.key === "Enter") {
-            e.preventDefault();
-            (e.target as HTMLTextAreaElement).blur();
-          }
-        }}
-        defaultValue={text}
-      />
+      {isImage ? (
+        <img
+          src={addProtocol(text)}
+          className="note-dialog__note-content--image"
```

Comment:
Why doesn't the image use the full width and height here? 
<img width="750" alt="Screenshot 2023-01-09 at 11 09 04" src="https://user-images.githubusercontent.com/36969812/211284315-5bc76fca-6c7d-4b4d-9685-500ef977f3a0.png">

Reply:
On the dashboard, the images are fully stretched over the note because it looks better. This however usually leads to some parts of the image being cut off. In order to see the full image, they aren't stretched in stack view.
This was discussed with the team in slack. Would you prefer it if they were filling the whole note in stack view as well?

Reply:
I think a larger view would look better on a lot of media, especially ones with text or images that are stretched horizontally. In focus mode, it's kinda difficult to make out what's displayed.

A: In normal view mode, this image is cut off. In focus mode, you can't read the text because it's too small.
<img width="872" alt="grafik" src="https://user-images.githubusercontent.com/7889564/218714377-8a8ac073-35a3-44ae-a927-867fa02f1114.png">
B: This cat would look a lot cuter if it was displayed in a larger size.
<img width="860" alt="grafik" src="https://user-images.githubusercontent.com/7889564/218714507-0c67359b-2ade-49e0-b297-41f8d8cfafc7.png">

In any case, the team already voted for this option, just wanted to add my two cents :)

Reply:
I kept it the way it was but added a simple functionality to zoom in on the pictures in stack view. Let me know what you think about it.

Reply:
I like the idea of being able to zoom into the picture by clicking on it, but I would implement it differently.  
Zooming into the center doesn't necessarily help to see the picture more clearly.
https://user-images.githubusercontent.com/7889564/219742741-d0009878-86ec-49d8-b778-e202293d3607.mov
In above example, zooming actually makes us see less content that's required to understand the meme (or whatever picture I just picked)

My suggestion would be to use imgurs approach, that is showing the media in **full size** and not in the container itself, but above everything else. Clicking anywhere on the screen results in it disappearing, or zooming out so to speak. That way, you would be able to see every detail of the picture, optionally.

Example:
https://imgur.com/gallery/TS0IEpP
![Bildschirm­foto 2023-02-17 um 19 11 48](https://user-images.githubusercontent.com/7889564/219745533-f240e166-08a9-4752-aaa6-371a1593ce1a.png)

As you can see, the bird takes full size of the screen while we want to look at it. Wdyt?

Reply:
Thanks for pointing that out. That's kinda the way I intended it to be, but it didn't work in Firefox which I didn't test. I fixed that and am now using a React portal to achieve the same effect as imgur. This is what it looks like:
https://user-images.githubusercontent.com/35272402/219949628-1d56f638-2b5c-4b0b-aa9f-ac7768dc87ce.mov
Is that good enough?

Reply:
Yes, I like it a lot 👍